pkg/sensors/exec/cgroups_test.go

```diff
@@ -1041,3 +1041,15 @@ func TestCgroupv1K8sHierarchyInHybridPids(t *testing.T) {
 func TestCgroupv1K8sHierarchyInHybridCpuset(t *testing.T) {
 	testCgroupv1K8sHierarchyInHybrid(t, "cpuset")
 }
+
+// This test will not use the blkio, it will fallback to the
+// best cgroup controller if available
+func TestCgroupv1K8sHierarchyInHybridBlkio(t *testing.T) {
+	testCgroupv1K8sHierarchyInHybrid(t, "blkio")
```

Comment:
pids and cpuset will be used if we specify them and if they are properly set which should be the case, as we want them to be a fallback in that order, so we explicitly test them.  For blkio we do not use it we fallback to the memory then pids or cpuset... that function will try to use the selected controller if available otherwise it falls back to the best one... meaning blkio or invalid are same we just ensure that we never fail.